src/GitLink/ArgumentParser.cs

```diff
@@ -45,7 +47,7 @@ public static Context ParseArguments(List<string> commandLineArguments, IProvide
                 return context;
             }
 
-            if (commandLineArguments.Count < 3)
+            if (commandLineArguments.Count < 3 && commandLineArguments.Count != 1)

```

Comment:
Not sure if this is a good way to check. I think we can safely remove this check and validate the Context instead (missing arguments will break the context).